owners/src/rules.js

```diff
@@ -34,6 +34,7 @@ class OwnersRule {
     this.filePath = ownersPath;
     this.dirPath = path.dirname(ownersPath);
     this.owners = owners;
+    this.priority = 1;
```

Comment:
For improved clarity, I think an enum should be used for the priorities, with names like `TOP_LEVEL_OWNER`, `WILDCARD_OWNER`, `PATTERN_OWNER`, etc. This will also make the tests easier to read and maintain.

Reply:
Done.